app/Http/Controllers/Compta/Bank/ShowBank.php

```
@@ -2,16 +2,363 @@
+        $object = new \Account($db);
+        $extrafields = new \ExtraFields($db);
+        $extrafields->fetch_name_optionals_label($object->table_element);
+        
+        return view('bank.create', [
+            'object' => $object,
+            'extrafields' => $extrafields,
+            'action' => 'create'
+        ]);
+    }
+    
+    private function edit(Request $request, int $id): View
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)